notificationdemo/source/android/.gitignore

```diff
@@ -0,0 +1,101 @@
+
+# Created by https://www.gitignore.io/api/android,intellij
```

Comment:
no need fr separate .gitignore in each project folder. This makes maintenance too complex due to code duplication.